features/cryptocell/FEATURE_CRYPTOCELL310/cc_internal.c

```diff
@@ -143,6 +143,8 @@ int convert_CrysError_to_mbedtls_err( CRYSError_t Crys_err )
     case CRYS_ECPKI_GEN_KEY_INVALID_PRIVATE_KEY_PTR_ERROR:
     case CRYS_ECPKI_EXPORT_PUBL_KEY_INVALID_PUBL_KEY_DATA_ERROR:
     case CRYS_ECPKI_BUILD_KEY_INVALID_PRIV_KEY_DATA_ERROR:
+    case CRYS_ECPKI_BUILD_KEY_INVALID_PRIV_KEY_SIZE_ERROR:
+    case CRYS_ECPKI_BUILD_KEY_INVALID_PUBL_KEY_SIZE_ERROR:
```

Comment:
What about `CRYS_ECPKI_BUILD_KEY_INVALID_PUBL_KEY_DATA_ERROR` and other `CRYS_ECPKI_.*_INVALID_.*_KEY_.*_ERROR`?

Reply:
I can add these, but since these aren't returned in any of our tests, there will be a full list of tests to add.

Reply:
Ok, so good enough for now.